app/src/main/java/eu/kanade/tachiyomi/data/track/suwayomi/SuwayomiApi.kt

```diff
@@ -25,26 +21,21 @@ import java.security.MessageDigest
 
 class SuwayomiApi(private val trackId: Long) {
 
-    private val network: NetworkHelper by injectLazy()
     private val json: Json by injectLazy()
 
-    private val client: OkHttpClient =
-        network.client.newBuilder()
-            .dns(Dns.SYSTEM) // don't use DNS over HTTPS as it breaks IP addressing
-            .build()
-
-    private fun headersBuilder(): Headers.Builder = Headers.Builder().apply {
-        if (basePassword.isNotEmpty() && baseLogin.isNotEmpty()) {
-            val credentials = Credentials.basic(baseLogin, basePassword)
-            add("Authorization", credentials)
-        }
+    private val sourceManager: SourceManager by injectLazy()
+    private val source: HttpSource by lazy {
```

Comment:
Have you checked what happens if you remove the source after enabling the tracker?

Reply:
Tested just now: Mihon seems to remove the tracking by itself, as the tracker becomes disabled